src/app/lightning/update-ln-payments.ts

```diff
@@ -19,8 +19,11 @@ export const updateLnPayments = async (): Promise<true | ApplicationError> => {
   for (const key of pubkeys) {
     const pubkey = key as Pubkey
 
-    let after: PagingStartToken | PagingContinueToken = undefined
-    while (processedLnPaymentsHashes.length < incompleteLnPayments.length) {
+    let after: PagingStartToken | PagingContinueToken | PagingStopToken = undefined
+    while (
+      processedLnPaymentsHashes.length < incompleteLnPayments.length &&
+      after !== false
```

Comment:
is this not the same of using only `after`

Reply:
`undefined` is a valid value for `after`, represents a paging start token. `false` is specifically used as the stop paging value (Alex does it as well in his [ln-accounting package](https://github.com/alexbosworth/ln-accounting/blob/master/records/get_all_payments.js#L115))